src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -35,7 +35,7 @@ public class AutoReviewController : Controller
         }
 
         [HttpPost]
-        public async Task<ActionResult> UploadAutoReview([FromForm] IFormFile file, string label, bool compareAllRevisions = false)
+        public async Task<ActionResult> UploadAutoReview([FromForm] IFormFile file, string label, bool compareAllRevisions = false, string packageVersion = null)
```

Comment:
would it be helpful to document in which cases the function expects `packageVersion` not to be `null`?

Reply:
I added it as null for now until all individual caller from language repo starts to send it. Once change is merged to individual repos, I will change this as required param.